contracts/RMRK/utils/RMRKNestableRenderUtils.sol

```diff
@@ -41,6 +42,36 @@ contract RMRKNestableRenderUtils {
         revert RMRKChildNotFoundInParent();
     }
 
+    /**
+     * @notice Used to retrieve the given child's index in its parent's pending child tokens array.
+     * @param parentAddress Address of the parent token's collection smart contract
+     * @param parentId ID of the parent token
+     * @param childAddress Address of the child token's colection smart contract
+     * @param childId ID of the child token
+     * @return The index of the child token in the parent token's pending child tokens array
+     */
+    function getPendingChildIndex(
+        address parentAddress,
+        uint256 parentId,
+        address childAddress,
+        uint256 childId
+    ) public view returns (uint256) {
+        IRMRKNestable.Child[] memory children = IRMRKNestable(parentAddress)
+            .pendingChildrenOf(parentId);
+        (parentId);
+        uint256 len = children.length;
+        for (uint256 i; i < len; ) {
+            if (
+                children[i].tokenId == childId &&
+                children[i].contractAddress == childAddress
+            ) return i;
+            unchecked {
+                ++i;
+            }
+        }
+        revert RMRKChildNotFoundInParent();
```

Comment:
Would it make sense for this to be:

```suggestion
revert RMRKPendingChildNotFoundInParent();
```

This could be understood in a way that the given child is not present in either the parent or the child token array. I agree that it can be inferred from context but still.. 🤔